.flake8

```diff
@@ -0,0 +1,4 @@
+[flake8]
+max-line-length = 120
+extend-ignore = W503
+inline-quotes = double
```

Comment:
Does this compel all quotes to be double quotes? If so, is this necessary?

Reply:
It's from this plugin: https://pypi.org/project/flake8-quotes/. I believe I linked you this before. It will require strings to use a certain quote type unless the other type could be used to reduce backslash escapes.

Reply:
I asked about it before, but I don't recall seeing a reply. 
Saw the page you linked, but this:

> Now you don’t need to worry about people like @sectioneight constantly complaining that you are using double-quotes and not single-quotes.

is a bit ambiguous as to scope of what the plugin does.

Reply:
It generates a flake warning if you use the wrong quote type unless using it reduces the number backslash escapes needed in the string.

Reply:
Yeah, I'd prefer not forcing a particular quote.

Reply:
I prefer having consistent quotes in a code base because there is no benefit to mixing them arbitrarily and it just makes the code look like a mess.